moirae/models/utils.py

```diff
@@ -0,0 +1,12 @@
+"""
+Collection of utility functions and classes for the models
+"""
+from .base import DegradationModel, HealthVariable
+
+
+class DummyDegradation(DegradationModel):
+    """
+    Class corresponding to no degradation, that is, the updated A-SOH is the same as the previous A-SOH.
+    """
+    def update_asoh(self, previous_asoh: HealthVariable, *args, **kwargs) -> HealthVariable:
+        return previous_asoh.model_copy(deep=True)
```

Comment:
Why are we copying here? That feels like an unnecessary computational overhead, especially if we do it every step.

Reply:
Ultimately, because we do not know what is going to be done with the A-SOH object returned. My original idea is that the `DegradationModel` always outputs a new `HealthVariable` object that represents a new state of the A-SOH, and it's up to whoever is using it to decide what to do with it and with the original A-SOH. I feel like this is a "better safe than sorry" type of overhead (kind of similar to the `SOCInterpolatedHealth` one), but I am open to changing it. I just feel like this particular overhead is relatively minor compared to the other things we are doing.

Reply:
That's fair. Returning a new copy makes sense to me